rs/boundary_node/rate_limits/canister/canister.rs

```diff
@@ -32,11 +31,26 @@ fn init(init_arg: InitArg) {
             state.set_authorized_principal(principal);
         });
     }
-    // Initialize an empty config corresponding to version=1
-    init_version_and_config(1);
+    with_canister_state(|state| {
+        if state.get_version().is_none() {
+            ic_cdk::println!("Initializing rate-limit config");
+            let current_time = ic_cdk::api::time();
+            init_version_and_config(current_time, state);
+        } else {
+            ic_cdk::println!("Rate-limit config is already initialized");
+        }
+    });
     // Spawn periodic job of fetching latest API boundary node topology
     // API boundary nodes are authorized readers of all config rules (including not yet disclosed ones)
     periodically_poll_api_boundary_nodes(init_arg.registry_polling_period_secs);
+    ic_cdk::println!("Finished canister init");
+}
+
+#[post_upgrade]
+fn post_upgrade(init_arg: InitArg) {
+    ic_cdk::println!("Starting canister post-upgrade");
+    init(init_arg);
+    ic_cdk::println!("Finished canister post-upgrade");
```

Comment:
Are all those logs really needed or are they just a left-over from your tests?

Reply:
It's rather "left-over" from the manual tests. Indeed, these logs are not available in production (one should use a different approach), but it can be a good idea to leave logs for test/debug builds. If somebody wants to deploy and test canister locally, these logs can be very helpful.

Reply:
In general, it's bad form to leave debug statements around. If someone needs to work on this locally and they need debug statements, they can always add those themselves temporarily. Alternatively, you could mark those as debug explicitely (i.e either use a dedicated `debug!` style function or add some `[debug]` prefix to those log lines.

Reply:
> Alternatively, you could mark those as debug explicitly.

Yes, i think this is the best approach. I will do it in a separate PR, if that's alright.

Reply:
@rikonor are You ok, to do this in the follow up PR? I would then rework all log statements at once.

Reply:
> are You ok, to do this in the follow up PR? I would then rework all log statements at once.
If you do it in a follow-up, it's fine with me 👍